### DISCUSSION TITLE
Ferramenta de mock para chamadas HTTP

### DISCUSSION THREAD

**Discussion by rpc1910** [diego3g/rsxp-2023#63] [General]
Fala pessoal, tudo beleza?

Irei trabalhar na issue de configuração de mocks para chamadas HTTP  #62  e até então não encontrei uma discussão sobre o assunto de qual ferramenta iremos utilizar.

Listo abaixo algumas ferramentas:

https://github.com/jaywcjlove/mocker-api
https://miragejs.com/
https://mswjs.io/

**Comment by weescoelho** [comment]
Acredito que o MirageJS seja uma boa pedida. O Fábio Vedovelli tem um starter-pack do Mirage para facilitar na configuração inicial. Podemos ganhar velocidade na implementação dele no projeto.

**Reply by rpc1910** [reply]
Boa, vou dar uma olhada @weescoelho !!

Obrigado

**Comment by rpc1910** [comment]
Bom, assim como o @diego3g  mencionou em seu comentário na discussão #2 , irei seguir na configuração do MirageJS.

Estarei fechando essa discussão para podermos focar nas demais abertas :)
